user.go

```diff
@@ -34,7 +35,10 @@ func (u *User) Mention() string {
 //             be added to the URL.
 func (u *User) AvatarURL(size string) string {
 	var URL string
-	if strings.HasPrefix(u.Avatar, "a_") {
+	if u.Avatar == "" {
+		discriminatorInt, _ := strconv.Atoi(u.Discriminator)
```

Comment:
Instead of throwing away this error, just use Discriminator as a string in endpoints.go

Reply:
Wasn't sure how much logic I should put into endpoints.go, I will change it 👍

Reply:
Endpoints shouldn't need any real logic, if you just use strings (it's kinda gross, but we don't use discriminators as ints anywhere else in the codebase)